modular_skyrat/master_files/code/datums/traits/neutral.dm

```diff
@@ -114,3 +114,12 @@
 				H.emote("laugh")
 				addtimer(CALLBACK(H, /mob/proc/emote, "laugh"), 5 SECONDS)
 				addtimer(CALLBACK(H, /mob/proc/emote, "laugh"), 10 SECONDS)
+
+/datum/quirk/feline_aspect
+	name = "Feline Traits"
```

Comment:
```suggestion
	name = "Felinid Traits"
```
dumbass

Reply:
felind is the neko race

feline is a name to refer to the members of the felidae family